wasm/jit/jit_amd64.go

```diff
@@ -297,16 +57,26 @@ type amd64Builder struct {
 	requireFunctionCallReturnAddressOffsetResolution []*obj.Prog
 }
 
-func (b *amd64Builder) compile() ([]byte, error) {
-	code, err := mmapCodeSegment(b.builder.Assemble())
+// Implements compiler.emitPreamble for amd64Compiler.
```

Comment:
I think godoc like this don't add value. Also IDEs warn when they don't start with the function name